manifest.json

```
@@ -1009,6 +1009,36 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm64",
+              "dockerfile": "src/ubuntu/21.10/helix/arm64v8",
+              "os": "linux",
+              "osVersion": "hirsute",
+              "tags": {
+                "ubuntu-21.10-helix-arm64v8-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
+              },
+              "variant": "v8"
+            }
+          ]
+        },
+        {
+          "platforms": [
+            {
+              "architecture": "amd64",
+              "dockerfile": "src/ubuntu/21.10/helix/amd64",
+              "os": "linux",
+              "osVersion": "hirsute",
```

```suggestion
              "osVersion": "impish",
```

https://github.com/dotnet/docker-tools/pull/900